### PR TITLE
CheckPriority.py

### PR DESCRIPTION
**Pull request by jiratQ**
**fix check priority for make it can Due with TieSituation**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-.env
+.env
```
Comment (phusitsom) [inline .gitignore:1]: Why?

**Review by phusitsom — COMMENTED** [review]
.

File: CheckPriority.py [hunk 2]
```diff
@@ -38,10 +38,14 @@ def winner(win: List[List[str]]):
...
+    playerwin=[]
+    for i in range len(player):
+        if(player[0][0][3]==player[i][0][3]):
+            playerwin.append(player[i])
     #return naame,howwin and card that use
-    return [player[0][1],player[0][0][1],player[0][0][3]]
-
-
+    #return [player[0][1],player[0][0][1],player[0][0][3]]
+    return playwerwin
```
Comment (phusitsom) [inline CheckPriority.py:47]: TYPO

File: CheckPriority.py [hunk 3]
```diff
@@ -38,10 +38,14 @@ def winner(win: List[List[str]]):
         player.append([best_card_ofone(win[i]),name])
     player.sort()
     player.reverse()
+    playerwin=[]
+    for i in range len(player):
```
Comment (phusitsom) [inline CheckPriority.py:42]: TYPO

**Comment by namirinz** [comment]
Why on help branch?

please change to features/poker

**Pull request closed without merging**